Add a tutorial demonstrating how to display plots in Optuna-Dashboard (#81)
optuna/optunahub: docs/source/tutorials/plot-in-optuna-dashboard.rst

to integrate and display plots generated by Optunahub modules in Optuna-Dashboard.

Preparation
-----------

First, ensure the necessary packages are installed by executing the following command in your terminal:

.. code-block:: console

    $ pip install optuna optunahub optuna-dashboard

Example
-------

We are going to use `Plot Pareto Front for Multiple Studies <https://hub.optuna.org/visualization/plot_pareto_front_multi/>`_  as example.

.. code-block:: python

    import optuna
    import optunahub
    from optuna_dashboard import save_plotly_graph_object

    def objective(trial: optuna.Trial) -> tuple[float, float]:
        x = trial.suggest_float("x", 0, 5)
        y = trial.suggest_float("y", 0, 3)

        v0 = 4 * x**2 + 4 * y**2
        v1 = (x - 5) ** 2 + (y - 5) ** 2
        return v0, v1

    samplers = [
        optuna.samplers.RandomSampler(),
        optuna.samplers.TPESampler(),
        optuna.samplers.NSGAIISampler(),
    ]
    studies = []
    for sampler in samplers:
        study = optuna.create_study(
            sampler=sampler,

>             study_name=f"{sampler.__class__.__name__}",

Reviewer: ```suggestion
            study_name=sampler.__class__.__name__,
```

            directions=["minimize", "minimize"],

>             storage=STORAGE_URL,

Reviewer: You need to provide what `STORAGE_URL` is.
Possibly, an SQLite3 URL could be provided so that users can run the code by copy&paste'ing the sample code.

        )
        study.optimize(objective, n_trials=50)
        studies.append(study)

    m = optunahub.load_module(
        "visualization/plot_pareto_front_multi"
    )
    fig = m.plot_pareto_front(studies)

>     save_plotly_graph_object(studies[0], fig)

Reviewer: Why not save the figure to all the studies?
```suggestion
    for study in studies:
        save_plotly_graph_object(study, fig)
```

> Execute this, then start the Optuna-Dashboard by running the following command:

Reviewer: ```suggestion
After running this code, launch the Optuna-Dashboard by the following command:
```

.. code-block:: console

>     $ optuna-dashboard STORAGE_URL

Reviewer: This `STORAGE_URL` is also not provided.

> Then, visit the page of the study you saved the plot.

Reviewer: If we save the figure to all the studies, it does not matter which study page we go to. 
```suggestion
Then, go to one of the study pages.
```
